cli/cmd/admin/start.go

```diff
@@ -106,7 +106,7 @@ type Config struct {
 // StartCmd starts an admin server. It only allows configuration using environment variables.
 func StartCmd(ch *cmdutil.Helper) *cobra.Command {
 	startCmd := &cobra.Command{
-		Use:   "start [jobs|server|worker]",
+		Use:   "start [jobs|server]",
```

Comment:
We still need both `jobs` and `worker` commands. When using `jobs`, it just triggers the configured jobs and exits. When using `worker`, it starts job workers that run forever and process jobs as they are submitted/scheduled. You can look at the old code and/or the Helm charts for details.